test/regress/expect/tpch0001_select/sql01.txt

```diff
@@ -0,0 +1,185 @@
+select 
+	c_name, 
+	(select 
+		count(O.o_orderkey) 
+	from 
```

Comment:
This is a non correlated subquery, which is efficiently handled by cached subquery(ie. only exec once), so subquery decorrelation shall not be applied here.

Reply:
got it

Reply:
So there should be some special branches to handle this in Expain()

Reply:
I solve the item in PR#264 "account subquery cost of scalarSubquery in SELECT clause to inclusive cost". This PR will unnested Subquery in SELECTION Clause. So, it may be discarded.